global: fix build failure

Same as:
* [invenio-records-ui #84](https://github.com/inveniosoftware/invenio-records-ui/issues/84)
* [invenio-records-rest #289](https://github.com/inveniosoftware/invenio-records-rest/issues/289)
* [invenio-search-ui #73](https://github.com/inveniosoftware/invenio-search-ui/issues/73)
* [invenio-previewer #111](https://github.com/inveniosoftware/invenio-previewer/issues/111)
* [invenio-theme #145](https://github.com/inveniosoftware/invenio-theme/issues/145)
* [invenio-base #147](https://github.com/inveniosoftware/invenio-base/issues/147)
* [invenio-i18n #81](https://github.com/inveniosoftware/invenio-i18n/issues/81)

https://travis-ci.org/inveniosoftware/invenio-previewer/jobs/656679537
```
ImportError while loading conftest '/home/travis/build/inveniosoftware/invenio-previewer/tests/conftest.py'.
tests/conftest.py:22: in <module>
    from flask_babelex import Babel
../../../virtualenv/python2.7.15/lib/python2.7/site-packages/flask_babelex/__init__.py:23: in <module>
    from werkzeug import ImmutableDict
E   ImportError: cannot import name ImmutableDict
The command "./run-tests.sh" exited with 4.
```

Fix also deprecation warning in Travis config:
https://travis-ci.org/inveniosoftware/invenio-previewer/jobs/656679537/config

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,8 +14,6 @@ addons:
 notifications:
   email: false
 
-sudo: false
-
 language: python
 
 cache:
@@ -27,8 +25,8 @@ env:
   - REQUIREMENTS=devel
 
 python:
-  - "2.7"
-  - "3.5"
+  - "3.6"
+  - "3.7"
 
 matrix:
   fast_finish: true
@@ -36,6 +34,7 @@ matrix:
     - env: REQUIREMENTS=devel
 
 before_install:
+  - "pip uninstall -y six"
   - "travis_retry pip install --upgrade pip setuptools py"
   # Bug in pkginfo wheel 1.3.0 (remove once fixed in pkginfo)
   - "travis_retry pip install --upgrade --no-binary=:all: pkginfo"
@@ -62,7 +61,8 @@ deploy:
   password:
     secure: "D0nJxy6c+YAIQCIy29cf0XkhzRA2zhI9TDFXi9GV+s5qcfgR8ds58CDgAdj/ErdvkWA5VBC13BS+668LWMtFIQENZ44Onc3J2UkiF8vHyzChu0FqtgCMT/qbFNeUIlJ6lwTsgAcWnQkh2jt3yTOvnbRuNdIKViyJpd6TWkKHl4khB7s4PWtyXsV7DshZhykAGqCFU2jizt1JDq2EuHNcvq79CjSwaGKdOBWWYzj6PalhOKae6/7bmg+4wQ+p0OwigIrI0irbes+xnVXEVmWalYTpLiLQaWmUrPKHpL0F6TVjFHPMYscFtp+d9Z+ROHAjsc5KN3uJmlrS6cGSHOVFxWBF6xnrJxWGd3hQboZXEvrzJno1R6pXTU7rh3/LINgXY+zS75GUPSIBvkHcvdlW6Ph8zTHE2FR67kUfFGv4ImwElMj2yt+B11n0DxdsKlSevduelnGLSeJWutow3VHetAIMXDBnHm1OYhOtqpHpmuulyX5tnbOcsDLnXPNX9GUUKZ3pUEjmINwWjfvEbV8fvfyPGl4gSsM7a+rtjlbX6CTA2GSPJaJl9LzBq5Ocnwt85iT/3IXtwiY4MEU3rwmNSI5SpTgP+eZfmrOw22ji0NpSxsJoTpcZ7gPij6C0DXmpnvBXQVjYPBLtFnTHBk11K2ZFAzYefcQprmdk8WIVEPU="
   distributions: "compile_catalog sdist bdist_wheel"
+  skip_existing: true
   on:
     tags: true
-    python: "2.7"
+    python: "3.6"
     condition: $REQUIREMENTS = release

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -10,6 +10,12 @@
 Changes
 =======
 
+Version 1.2.0 (Release 2020-03-13)
+----------------------------------
+
+- Change flask dependency to centrally managed by invenio-base
+- Drop support for Python 2.7
+
 Version 1.1.0 (Release 2019-12-20)
 ----------------------------------
 

--- a/invenio_previewer/version.py
+++ b/invenio_previewer/version.py
@@ -14,4 +14,4 @@ and parsed by ``setup.py``.
 
 from __future__ import absolute_import, print_function
 
-__version__ = "1.1.0"
+__version__ = "1.2.0"

--- a/requirements-devel.txt
+++ b/requirements-devel.txt
@@ -8,3 +8,4 @@
 
 -e git+https://github.com/inveniosoftware/invenio-files-rest.git#egg=invenio-files-rest
 -e git+https://github.com/inveniosoftware/invenio-records-ui.git#egg=invenio-records-ui
+-e git+https://github.com/inveniosoftware/invenio-base.git#egg=invenio-base

--- a/setup.py
+++ b/setup.py
@@ -51,12 +51,12 @@ setup_requires = [
 
 install_requires = [
     'cchardet>=1.0.0',
-    'Flask-BabelEx>=0.9.3',
-    'Flask>=0.11.1',
     'invenio-assets>=1.1.2',
+    'invenio-base>=1.2.2',
     'invenio-formatter>=1.0.2',
-    'invenio-pidstore>=1.1.0',
-    'invenio-records-ui>=1.0.1',
+    'invenio-i18n>=1.2.0',
+    'invenio-pidstore>=1.2.0',
+    'invenio-records-ui>=1.1.0',
     'ipython>=4.1.0',
     'mistune>=0.7.2',
     'nbconvert[execute]>=4.1.0',

